volume/mounts/lcow_parser.go

```diff
@@ -7,6 +7,23 @@ import (
 	"github.com/docker/docker/api/types/mount"
 )
 
+// NewLCOWParser creates a parser with Linux Containers on Windows semantics.
+func NewLCOWParser() Parser {
+	return &lcowParser{
+		windowsParser{
+			fi: defaultFileInfoProvider{},
+		},
+	}
+}
+
+// rxLCOWDestination is the regex expression for the mount destination for LCOW
+//
+// Destination (aka container path):
+//    -  Variation on hostdir but can be a drive followed by colon as well
+//    -  If a path, must be absolute. Can include spaces
+//    -  Drive cannot be c: (explicitly checked in code, not RegEx)
+const rxLCOWDestination = `(?P<destination>/(?:[^\\/:*?"<>\r\n]+[/]?)*)`
```

Comment:
This isn't new, so I'm not suggesting we change it now, but shouldn't this (and the friends it moved away from) be `regexp.MustCompile`'d?

Reply:
Yes, probably makes sense to do so (help me remind in case I forget)

Reply:
https://github.com/moby/moby/issues/42667 :smirk: :innocent: